docs/PROMPT_TEMPLATE_LANGUAGE.md

```diff
@@ -1,4 +1,4 @@
-Prompts are the inputs or queries that a user or a program gives to an LLM AI,
+Prompts are the inputs or queries that a user or a program gives to an Large Learning Model ([LLM](https://en.wikipedia.org/wiki/Wikipedia:Large_language_models)) AI,
```

Comment:
LLM stands for Large Language Model.

https://en.wikipedia.org/wiki/Large_language_model Might be a more appropriate link.